README.md

```diff
@@ -83,6 +84,23 @@ export PASS="1"
 ./openvpn-install.sh
 ```
 
+### Split-tunnel usage
+If you only want to tunnel specific subnets through the VPN-tunnel, you can enable Split-tunnel configuration by passing the TUNNEL_CIDR_BLOCKS environment variable. For example, if you want to tunnel just the subnets 10.0.1.0 with netmask 255.255.255.0 and 10.0.2.0 with netmask 255.255.255.0, you specify 10.0.1.0/24 and 10.0.2.0/24:
+
+On the command line:
+```bash
+TUNNEL_CIDR_BLOCKS=(10.0.1.0/24 10.0.2.0/24) ./openvpn-install.sh
```

Comment:
Why parenthesis instead of double quotes?

Reply:
I think that was a zsh-thing. I'll check it out soon. Thanks for noticing.

Reply:
Update: @randomshell this is the proper way to declare an array in both bash as zsh, as far as I know.